native/mod_manager/node.c

```diff
@@ -275,3 +275,37 @@ mem_t * create_mem_node(char *string, int *num, int persist, apr_pool_t *p, slot
 {
     return(create_attach_mem_node(string, num, CREATE_SLOTMEM|persist, p, storage));
 }
+
+
+/**
+ * find the first node correspond to host/port.
+ * @param pointer to the shared table.
+ * @param node node to read from the shared table.
+ * @return address of the read node or NULL if error.
```

Comment:
That's misleading. Better would be 

> @return `APR_SUCCESS` in case the node is found and its address is in `data`, `APR_NOTFOUND` otherwise

or something like that

Reply:
fixed